css-grid-classes/styles.css

```diff
@@ -0,0 +1,39 @@
+.container {
+  font-family: sans-serif;
+  box-sizing: border-box;
+  width: 44%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
```

Comment:
You shouldn't center all content in each row, because you may one day want a row that doesn't have its content centered. The name of the game with grid systems is doing whatever styling that one tool needs, and only that styling, nothing unique to a given situation